entrypoint.sh

```diff
@@ -157,7 +158,25 @@ main() {
   bindplane apply "$destination_path"
 
   echo "Applying configuration path: $configuration_path"
-  bindplane apply "$configuration_path"
+  bindplane apply "$configuration_path" > configuration.out
+  cat configuration.out
```

Comment:
Capture the output in configuration.out to be used later. Cat it out so it is still visable in the action output.